example/README.md

```diff
@@ -0,0 +1,78 @@
+
+# FluentD to Dynatrace Log Ingest
```

Comment:
Please link to the example folder from the top-level README so it's easier to discover.